apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,219 @@
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <CTABanner.ButtonGroup>
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </CTABanner.ButtonGroup>
+</CTABanner>
+```
+
+### Optional shadow
+
+The shadow can be removed using the `hasShadow` prop on the root `CTABanner`. This will render the component without the background shadow.
+
+```jsx live
+<CTABanner hasShadow={false}>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <CTABanner.ButtonGroup>
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </CTABanner.ButtonGroup>
+</CTABanner>
+```
+
+### Shadow color customization
+
+The default colors can be customized through dedicated CSS variables.
+
+| Name                                   | Default Light                    | Default Dark                     |
+| :------------------------------------- | :------------------------------- | :------------------------------- |
+| `--brand-CTABanner-shadow-color-start` | `--base-color-scale-green-5-hsl` | `--base-color-scale-green-2-hsl` |
+| `--brand-CTABanner-shadow-color-end`   | `--base-color-scale-blue-5-hsl`  | `--base-color-scale-blue-3-hsl`  |
+
+```javascript live noinline
+const App = () => {
+  const designTokenOverrides = `
+  .custom-colors[data-color-mode='dark'] {
+
+    /*
+     * Modify the value of these tokens.
+     * Remember to apply light mode equivalents if you're enabling theme switching.
+     */
+    --brand-CTABanner-shadow-color-start: var(--base-color-scale-purple-5-hsl);
+    --brand-CTABanner-shadow-color-end: var(--base-color-scale-red-5-hsl);
+
+  }
+`
+  return (
+    <ThemeProvider colorMode="dark" className="custom-colors">
+      <style>{designTokenOverrides}</style>
+      <CTABanner>
+        <CTABanner.Heading>
+          Where the most ambitious teams build great things
+        </CTABanner.Heading>
+        <CTABanner.Description>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+          ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+          elementum.
+        </CTABanner.Description>
+        <CTABanner.ButtonGroup>
+          <Button>Primary Action</Button>
+          <Button>Secondary Action</Button>
+        </CTABanner.ButtonGroup>
+      </CTABanner>
+    </ThemeProvider>
+  )
+}
+
+render(<App />)
+```
+
+## Component props
+
+### CTABanner <Label>Required</Label>
```

Comment:
My previous comments about this seem to have disappeared 😕 

This is breaking in the rendered version because Label isn't imported.

fix is - from memory (try it locally) - to add `import {Label} from "@primer/react"` at the top of the file. 

![Screenshot 2022-12-07 at 13 38 53](https://user-images.githubusercontent.com/13340707/206194145-7a943e61-2002-44c2-88f8-b240c76f5949.png)

Reply:
Ahh I thought I fixed this but I imported it from primer-brand which didn't make sense.

Reply:
This will be in the next push